docs/graphql-api.md

```diff
@@ -183,6 +201,21 @@ The GraphQL spec defines `Int` and `Float` to have sizes of 32 bytes.
 
 This API additionally includes a `BigInt` number type to represent arbitrarily large integer numbers.
 
+## 3.4 Enums 
```

Comment:
This is not part of https://github.com/graphprotocol/specs/tree/master/graphql-api but I think that's ok. I suspect we'll move these docs into their own repo soon anyway.